subset/switches/test_switch

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+# Setup for accessing control plane switch. If LOCAL_IP is defined, which
+# is the intended local address for this node on the control plane then
+# SWITCH_IP will be the IP address of the OpenFlow switch.
+if [ -n "$LOCAL_IP" ]; then
+    echo Configuring network with local address $LOCAL_IP.
+    ip addr add $LOCAL_IP dev `hostname`-eth0
+    echo Switch test with port $SWITCH_PORT.
+    ping -n -c 10 $SWITCH_IP
+    java -jar switches/target/switchtest-0.0.1-jar-with-dependencies.jar $SWITCH_IP $SWITCH_PORT
+else
+    echo LOCAL_IP not configured, assuming no network switch.
+fi
+
+cp -r tmp/report.txt /tmp/report.txt
```

Comment:
What does the report look like when it passes?  Is it trivial to distinguish "test pass" from "test did not run because switch not configured"?  Want to make sure we avoid false positives.